public/Sjofn/practice/sjofn4-5.html

```diff
@@ -0,0 +1,52 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Practice 4</title>
+</head>
+<body>
+</body>
+<script>
+
+    function sayHello(name) {
+        console.log(`Hello ${name}!`);
+    }
+
+    function generateRandomInt() {
+        let randomNum = Math.floor(Math.random()*10);
+        console.log(`Random integer: ${randomNum}`);
+        return randomNum;
+    }
+
+    function isEven(a) {
+        return a % 2 === 0;
+
+    }
+
+    let yourGeneratedNumber = generateRandomInt();
+    console.log(`${yourGeneratedNumber} is ${isEven(yourGeneratedNumber) ? 'Even' : 'Odd'}`);
+
+    function generateRandomIntInRange(min, max) {
+        return Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
According to the task, the function should also print some text in the console. Pleasem check.